### PR TITLE
Update package manifest and property editor docs

### PR DESCRIPTION
**Pull request by alina-tincas**
## Description

_What did you add/update/change?_
Update package manifest and property editor docs as they were outdated from v8:
- https://docs.umbraco.com/umbraco-cms/extending/property-editors/package-manifest 
- https://docs.umbraco.com/umbraco-cms/tutorials/creating-a-property-editor - also tested until step 2. There might be some code that is obsolete in step 3 and 4 but with v14 where angular is not used anymore these tutorial might need to be revisited and re-tested
## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
CMS 10, 12, 13

## Deadline (if relevant)

_When should the content be published?_
anytime

### CODE REVIEW

File: 10/umbraco-cms/extending/property-editors/package-manifest.md [hunk 1]
```diff
@@ -91,7 +94,7 @@ The manifest can contain seven root collections, none of them are mandatory:
 
 ### Telemetry elements
 
-In version 9.2 some additional root elements was added, the purpose of these are to control and facilitate telemetry about the package, none of these are mandatory. The properties are:
+From version 9.2 some additional root elements was added, the purpose of these are to control and facilitate telemetry about the package, none of these are mandatory. The properties are:
```
Comment (jonat123) [inline 10/umbraco-cms/extending/property-editors/package-manifest.md:97]:
```suggestion
From version 9.2 additional root elements were added, the purpose of these is to control and facilitate telemetry about the package, none of these are mandatory. The properties are:
```

File: 10/umbraco-cms/tutorials/creating-a-property-editor/README.md [hunk 2]
```diff
@@ -4,11 +4,7 @@ description: A guide to creating a property editor in Umbraco
 
 # Creating a Property Editor
 
-{% hint style="warning" %}
-This tutorial has last been last tested on v8 and therefore might not work as intended for v10 and above.
-{% endhint %}
-
-This guide explains how to set up a property editor, hook it into Umbraco's Data Types, AngularJS' modules, and its injector, and finally how we can test our property editor.
+This guide explains how to set up a property editor, hook it into Umbraco's Data Types, AngularJS modules, and its injector, and finally how we can test our property editor.
```
Comment (jonat123) [inline 10/umbraco-cms/tutorials/creating-a-property-editor/README.md:7]:
```suggestion
This guide explains how to set up a property editor and hook it into Umbraco's Data Types, AngularJS modules, and injector. Finally, it explains how we can test our property editor.
```

File: 10/umbraco-cms/tutorials/creating-a-property-editor/README.md [hunk 3]
```diff
@@ -27,12 +23,16 @@ This tutorial covers how to use AngularJS with Umbraco, so it does not cover Ang
 
 ## The End Result
 
-By the end of this tutorial, we will have a suggestion data type running inside of Umbraco, registered as a Data Type in the backoffice, and assigned to a Document Type. The data type can create and suggest values.
+By the end of this tutorial, we will have a "suggestions" data type running inside of Umbraco, registered as a Data Type in the backoffice, and assigned to a Document Type. The data type can create and suggest values.
```
Comment (jonat123) [inline 10/umbraco-cms/tutorials/creating-a-property-editor/README.md:26]:
```suggestion
By the end of this tutorial, we will have a "suggestions" data type running inside of Umbraco. It is registered as a Data Type in the backoffice, and assigned to a Document Type. The data type can create and suggest values.
```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@alina-tincas Looks good :) 

I have made some suggestions to fix the vale errors :)

Please make sure they are added to all the versions 😄

**Review by eshanrnh — COMMENTED** [review]
Looks good to me. Couple of suggestions though 🙂

File: 10/umbraco-cms/extending/property-editors/package-manifest.md [hunk 4]
```diff
@@ -15,62 +8,72 @@ This is a sample manifest, it is always stored in a folder in `/App_Plugins/{You
...
 ```
 
 ## Sample Manifest with Csharp
 
-You can also register your files by implementing a `IManifestFilter` instead of creating a `package.manifest`. Create a `ManifestFilter.cs` file and implement the `IManifestFilter` interface.
+You can also register your files by implementing a `IManifestFilter` instead of creating a `package.manifest`. Create a `ManifestFilter.cs` file and implement the `IManifestFilter` interface. Then define the composer using the `IComposer` interface.
 
 ```csharp
-public class ManifestFilter : IManifestFilter
+namespace MyProject
```
Comment (eshanrnh) [inline 10/umbraco-cms/extending/property-editors/package-manifest.md:39]: using Umbraco.Cms.Core.Composing;

File: 10/umbraco-cms/extending/property-editors/package-manifest.md [hunk 5]
```diff
@@ -15,62 +8,72 @@ This is a sample manifest, it is always stored in a folder in `/App_Plugins/{You
...
     "version": "1.0.0 beta",
     "allowPackageTelemetry": true,
     "bundleOptions": "Default",
-    "packageView": "/App_Plugins/SirTrevor/SirTrevor-config.html",
+    "packageView": "/App_Plugins/Suggestions/suggestion-config.html",
     "propertyEditors": [
         {
-            "alias": "Sir.Trevor",
-            "name": "Sir Trevor",
+            "alias": "Suggestions",
```
[the remark this answers is not in the document]
Comment (alina-tincas) [reply 10/umbraco-cms/extending/property-editors/package-manifest.md:18]: not necessarily. It works with both small and big first letter.

File: 10/umbraco-cms/extending/property-editors/package-manifest.md (hunk 4 above)
Comment (alina-tincas) [reply 10/umbraco-cms/extending/property-editors/package-manifest.md:39]: there are 2 more using statements apart from composing, I will add them as well

**Comment by alina-tincas** [comment]
Thank you both for the feedback 💪 I have made the changes as per your feedback, so I will merge this in 😊

**Pull request merged**